modules/microshift-lvm-thin-volumes.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * microshift_storage/creating-volume-snapshots-microshift.adoc
+
+:_content-type: CONCEPT
+[id="microshift-lvm-thin-volumes_{context}"]
+= About LVM thin volumes
+
+To use advanced storage features such as creating volume snapshots or volume cloning, you must provision a logical volume manager (LVM) thin-pool on the {op-system-ostree} host and configure both the logical volume manager storage (LVMS) provider and the cluster. LVM thin-pools must be attached to a volume group.
+
+[IMPORTANT]
+====
+To create CSI snapshots, you must configure thin volumes on the {op-system-ostree} host. The CSI does not support volume shrinking because of the potential for data loss.
+====
+
+For LVMS to manage thin logical volumes (LVs), a thin-pool `device-class` must be specified in the `lvmd.yaml` file. Multiple thin-pool device classes are permitted.
+
+.Example `lvmd.yaml` that specifies a single device class for a thin-pool
+
+[source, yaml]
+----
+socket-name: <1>
+device-classes: <2>
+  - name: thin <3>
+    default: true
+    spare-gb: 0 <4>
+    thin-pool:
+      name: thin
+      overprovision-ratio: 10 <5>
+    type: thin
+    volume-group: ssd <6>
+    stripe: <7>
+    stripe-size: <8>
+    lvcreate-options: <9>
+----
+[.small]
+<1> String. The UNIX domain socket endpoint of gRPC. Defaults to  '/run/lvmd/lvmd.socket'.
+<2> A list of maps for the settings for each `device-class`.
+<3> String. The name of the `device-class`. Thin provisioning is required to create volume snapshots.
```

Comment:
The name isn't significant, it just has to be unique. The fact that the name in the example is "thin" is just to differentiate it from other device classes. So the statement here about "Thin provisioning is required..." is probably better attached to a footnote for line 30, where the `thin` flag is actually set.